rollup-sequencer/package.json

```diff
@@ -19,8 +19,8 @@
     "typescript": "5.3.2"
   },
   "dependencies": {
-    "@mangata-finance/sdk": "2.2.1-eth-rollup-develop.54",
-    "@mangata-finance/types": "2.2.3-eth-rollup-develop.61",
+    "gasp-sdk": "0.0.2-eth-rollup-develop.0",
```

Comment:
I see in this [pipeline](https://github.com/mangata-finance/gasp-dev-kit/actions/runs/9416370307) that we have versions: 
gasp-types@0.0.2-eth-rollup-develop.1
gasp-sdk@0.0.2-eth-rollup-develop.1
Is there a reason why you used the versions with .0 at the end?